cmd/daemon/main.go

```diff
@@ -19,22 +19,28 @@ package main
 import (
 	"flag"
 	"math/rand"
+	"net/http"
+	_ "net/http/pprof"
 	"time"
 
-	"github.com/openkruise/kruise/pkg/client"
 	"github.com/openkruise/kruise/pkg/daemon"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
+
+	"github.com/openkruise/kruise/pkg/client"
 	"github.com/openkruise/kruise/pkg/features"
 	utilfeature "github.com/openkruise/kruise/pkg/util/feature"
 	"github.com/spf13/pflag"
 	"k8s.io/klog/v2"
 	"k8s.io/klog/v2/klogr"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
-	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
 )
 
 var (
-	bindAddr = flag.String("addr", ":10221", "The address the metric endpoint and healthz binds to.")
+	bindAddr    = flag.String("addr", ":10221", "The address the metric endpoint and healthz binds to.")
+	pprofAddr   = flag.String("pprof-addr", ":10222", "The address the pprof binds to.")
+	metricsAddr = flag.String("metrics-addr", ":10223", "The address the metric endpoint binds to.")
```

Comment:
The metrics server is already included in `bindAddr`.

Reply:
> The metrics server is already included in `bindAddr`.

it has removed in latest code.